uftrace.c

```diff
@@ -309,6 +309,15 @@ static uint64_t parse_time(char *arg)
 	return val;
 }
 
+static bool is_time_unit(const char *str)
+{
+	/* <time_unit>  :=  "ns" | "us" | "ms" | "s" */
+	if (str[strlen(str) - 1] == 's')
+		return true;
```

Comment:
Note that time unit can be "usec" and so on.  I think it's better to use `isalpha()`.  Also I think the name should be "has_time_unit"..